mangaki/mangaki/choices.py

```diff
@@ -38,3 +40,36 @@
     ("authors", "Auteurs"),
     ("composers", "Compositeurs"),
 )
+
+
+class Category:
+    CHOICES = OrderedDict([
+        (1, "Anime"),
+        (2, "Manga"),
+        (3, "Album"),
+    ])
+    PARAMS = {
+        1: {'slug': 'anime'},
+        2: {'slug': 'manga'},
+        3: {'slug': 'album'},
+    }
+    _id_of_slug = {v['slug']: k for k, v in PARAMS.items()}
```

Comment:
Si ça paraît plus complexe parce qu'à la fin tu dois dire: enums → functional black magic à la map / filter / blabla → ordered dict, it's fine this way!